src/expression.js

```diff
@@ -274,7 +274,12 @@ Call.prototype.value = function(scope, helperScope, helperOptions){
 };
 
 Call.prototype.closingTag = function() {
-	return this.methodExpr.key.slice(1);
+	//!steal-remove-start
+	if(this.methodExpr[canSymbol.for('can-stache.originalKey')]) {
+		return this.methodExpr[canSymbol.for('can-stache.originalKey')];
+	}
+	//!steal-remove-end
+	return this.methodExpr.key;
```

Comment:
does this not need to strip off the first character anymore?

Reply:
the first character that was being stripped was the `@` from syntaxes like `{{#todos}}{{/todos}}` (where the key was `@todos`). If you used something like `{{#data.list}}{{/data.list}}`, key would be `data@list`, which was incorrectly stripped before.